.github/workflows/ghc.yml

```diff
@@ -382,3 +404,18 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           folder: dist
           single-commit: true
+
+      - name: Copyrights
+        - uses: actions/checkout@v4
```

Comment:
Move this and the following step into the `check` job. No need for a new `Copyrights` job.